downstream/modules/aap-hardening/con-network-firewall-services-planning.adoc

```diff
@@ -3,20 +3,24 @@
 
 [id="con-network-firewall-services_{context}"]
 
-= Network, firewall, and network services planning for {PlatformNameShort}
+//= Network, firewall, and network services planning for {PlatformNameShort}
 
-[role="_abstract"]
+//[role="_abstract"]
 
-{PlatformNameShort} requires access to a network to integrate to external auxiliary services and to manage target environments and resources such as hosts, other network devices, applications, cloud services. The link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/red_hat_ansible_automation_platform_planning_guide/index#ref-network-ports-protocols_planning[network ports and protocols] section of the {PlatformNameShort} planning guide describes how {PlatformNameShort} components interact on the network as well as which ports and protocols are used, as shown in the following diagram:
+//{PlatformNameShort} requires access to a network to integrate to external auxiliary services and to manage target environments and resources such as hosts, other network devices, applications, cloud services. 
+//The link:{URLPlanningGuide}?ref-network-ports-protocols_planning[network ports and protocols] section of {TitlePlanningGuide} describes how {PlatformNameShort} components interact on the network as well as which ports and protocols are used, as shown in the following diagram:
 
-.{PlatformNameShort} Network ports and protocols
-image::aap-network-ports-protocols.png[Interaction of {PlatformNameShort} components on the network with information about the ports and protocols that are used.]
+//.{PlatformNameShort} Network ports and protocols
+//image::aap-network-ports-protocols.png[Interaction of {PlatformNameShort} components on the network with information about the ports and protocols that are used.]
 
-When planning firewall or cloud network security group configurations related to {PlatformNameShort}, see the link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html-single/red_hat_ansible_automation_platform_planning_guide/index#ref-network-ports-protocols_planning[Network ports and protocols] section of the {PlatformNameShort} Planning Guide to understand what network ports need to be opened on a firewall or security group.
+When planning firewall or cloud network security group configurations related to {PlatformNameShort}, see the 
+'Network Ports' section of your chosen topology in link:{LinkTopologies}
```

Comment:
I think _'Network Ports'_ should be formatted as follows: "Network ports" https://www.ibm.com/docs/en/ibm-style?topic=punctuation-quotation-marks